modules/bidi/assets/items/textfile.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is there any reason to have this empty file?

Reply:
No specific reason for text file. I was just learning how to use the editor.
Updated the icon file as per required format and pushed it using git command. 
Thanks for your help

P.S I was just learning how to use the web loader since I had used git command line before for other completed tasks :)